src/System.Private.ServiceModel/tests/Scenarios/Binding/Custom/CustomBindingTests.cs

```diff
@@ -12,7 +12,44 @@
 
 public class CustomBindingTests : ConditionalWcfTest
 {
-    // Client and Server bindings setup exactly the same using default settings.
+    // Http: Client and Server bindings setup exactly the same using default settings.
```

Comment:
Reviewers: this test is just a duplicate of the Https test that was already here (below).  It differs in using Http rather than Https and therefore does not depend on a [ConditionalFact].